tools/Makefile

```diff
@@ -28,8 +28,17 @@ FLASH_USED_END = $$((0x`$(OBJDUMP) -t ../app/.output/eagle/debug/image/eagle.app
 
 all:	spiffsscript
 
-spiffsscript: remove-image
-	$(MAKE) -C spiffsimg CC=gcc
+.PHONY: spiffsimg
+
+.PHONY: spiffsimg/spiffsimg
+
+spiffsimg: spiffsimg/spiffsimg
+	@echo Built spiffsimg in spiffsimg/spiffsimg
+
+spiffsimg/spiffsimg: 
+	@$(MAKE) -C spiffsimg CC=gcc
```

Comment:
Do we have a $(HOSTCC) defined? Should we? CC=$(HOSTCC) would be clearer, and also allow for building with host clang if so specified on the command line.